js/range-filter.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+var $ = require('jquery');
+var Filter = require('./filter-base');
+
+function RangeFilter(elm) {
+  Filter.Filter.call(this, elm);
+  this.id = this.$input.attr('id');
+  this.$submit = this.$elm.find('button');
```

Comment:
Submit button isn't being found because the button element isn't within `.js-filter` in 18F/openfec-web-app#1540